### PR TITLE
issue #9 fix

### PR DESCRIPTION
**Pull request by PeonMike**

### CODE REVIEW

**Comment by skhokhlov** [comment]
We have problem with merge

File: src/tms.cpp [hunk 1]
```diff
@@ -392,14 +392,14 @@ void GOLO2(double *quasi, vector<vector<double>> &bounds) {
...
+double func(vector< double > &p) {
     return p[0] * p[0] + p[1] * p[1];
 }
 
 //Функция сравнения точек для сортировки
 int cmp(const void *a, const void *b) {
-    auto *x = (struct point *) a;
-    auto *y = (struct point *) b;
+    struct point *x = (struct point *) a;
+    struct point *y = (struct point *) b;
```
Comment (skhokhlov) [inline src/tms.cpp]: You have problem with merge

File: src/tms.cpp [hunk 2]
```diff
@@ -413,8 +413,10 @@ m_ - колличество минимальныйх точек для запу
 bounds - границы области
 */
 
-void GENIN2(unsigned long dimen_, unsigned long seqlen_, unsigned dots_, double (*fun)(vector<double> &),
-       vector<vector<double>> &bounds) {// PROGRAM
+void
+GENIN2(unsigned long dimen_, unsigned long seqlen_, unsigned dots_, double (*fun)(vector< double > &),
+       vector< vector< double > > &bounds) // PROGRAM
+{
```
Comment (skhokhlov) [inline src/tms.cpp]: Yet another problem

**Comment by Arquestro** [comment]
Pull the origin/develop

File: src/tms.cpp [hunk 3]
```diff
@@ -42,11 +42,11 @@ int nextq2[MAX_DIMENSION];
...
 const int MAX_Q_VALUE = 50;
 const int MAX_POLY_DEGREE = 50, CURRENT_DEGREE_VALUE = 0;
-int p, q, add[MAX_Q_VALUE/*!*/][MAX_Q_VALUE/*!*/], mul[MAX_Q_VALUE/*!*/][MAX_Q_VALUE/*!*/],
-        sub[MAX_Q_VALUE/*!*/][MAX_Q_VALUE/*!*/];
+int p, q, add[MAX_Q_VALUE/*!*/][MAX_Q_VALUE/*!*/], mul[MAX_Q_VALUE/*!*/][MAX_Q_VALUE/*!*/], sub[MAX_Q_VALUE/*!*/][MAX_Q_VALUE/*!*/];
 
-vector<vector<double>> vResult;
-vector<double> vector_;
+
+vector< vector< double > > vResult;
```
Comment (Arquestro) [inline src/tms.cpp]: Templates that dont include template types, can be written without braces

**Review by Arquestro — COMMENTED** [review]
You have to pull origin, or youll erase all changes that have been done since you've branched out

**Review by skhokhlov — CHANGES_REQUESTED** [review]
You should merge latest changes

**Comment by PeonMike** [comment]
все пофиксил

**Pull request merged**